rules/falco_rules.yaml

```diff
@@ -397,7 +397,10 @@
 
 - rule: Disallowed SSH Connection
   desc: Detect any new ssh connection to a host other than those in an allowed group of hosts
-  condition: (inbound_outbound) and ssh_port and not allowed_ssh_hosts
+  condition: > 
```

Comment:
no idea if it exists but what about using a sort of `yaml-linter` to do that automatically, Without some enforcement it will be hard to respect the style in future PRs

Reply:
Great idea! https://github.com/falcosecurity/rules/issues/144